README.md

```diff
@@ -9,20 +12,22 @@ This CDK library automatically configures ingestion of metrics, traces, and logs
 - Managing subscriptions from the Datadog Forwarder to your Lambda function log groups.
 
 ## Installation
-
-Download the code and import datadog-cdk-constructs/lib/index.ts.
-
+```
+yarn add --dev datadog-cdk-constructs
+# or
+npm install datadog-cdk-constructs --save-dev
+```
 ## Usage
 
 ### AWS CDK
 
-import datadog-cdk-constructs/lib/index.ts.
+import datadog-cdk-constructs/dist/lib/index.js.
```

Comment:
I don't think customers should have to use this full path. Maybe there is something you can add to the package.json to avoid this...